packages/assets-controllers/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [73.1.0]
```

Comment:
should this be a patch ? I see we have nothing new compared to previous version only some fixes ?